qt/editor_dialog.cpp

```diff
@@ -56,10 +56,11 @@ EditorDialog::EditorDialog(QWidget * parent, osm::EditableMapObject & emo)
     int namesRow = 0;
     for (auto const & ln : emo.GetNamesDataSource().names)
     {
-      namesGrid->addWidget(new QLabel(ln.m_lang), namesRow, 0);
+      namesGrid->addWidget(new QLabel(QString::fromUtf8(ln.m_lang.data(), ln.m_lang.size())), namesRow, 0);
```

Comment:
Is it the best way to make QString here and below?
Ok, simple ```std::string(ln.m_lang)``` or even ```ln.m_lang.c_str()```;

Reply:
1. Temporary string is overhead.
2. There is no c_str() in string_view.
3. It is the best and the fastest way, what worries you?

Reply:
Ok, no problem here.